### PR TITLE
Add Bug Fix PR Template

### PR DESCRIPTION
**Pull request by hiro-daikin**
This adds an RCA style PR template for bug fixes

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/bug_fixes.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+<!-- Explain the fix implemented to address the root cause of the problem. -->
+
+## How Has This Been Tested?
+
+<!-- Describe what automated tests you added or updated to verify the bug fix. -->
+
+### Regression Tests
+
+- Tests covering the [describe specific functionality or edge cases] have been updated/added.
+- Verified that the bug does not reoccur under [specific scenarios].
```
Comment (yodas7) [inline .github/PULL_REQUEST_TEMPLATE/bug_fixes.md:33]: These sections seemed to overlap. Is there a difference?

File: .github/PULL_REQUEST_TEMPLATE/bug_fixes.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+### Manual Testing
+
+- Verified the fix manually by [describe the manual testing process].
+- Ensured that the issue no longer occurs in the following states: [Normal; Hover; Pressed; Disabled], if applicable.
```
Comment (yodas7) [inline .github/PULL_REQUEST_TEMPLATE/bug_fixes.md:38]: I'm not sure whether we should include a Manual Testing section.
Ideally, tests should be automated to avoid future regression.

**Review by yodas7 — COMMENTED** [review]
Thank you.
I made some comments about the testing.

File: .github/PULL_REQUEST_TEMPLATE/bug_fixes.md (hunk 2 above)
Comment (rinjInTokyo) [reply .github/PULL_REQUEST_TEMPLATE/bug_fixes.md:38]: Should we Investigation and fix the similar bug?

File: .github/PULL_REQUEST_TEMPLATE/bug_fixes.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+<!-- Fill in the component or bug name below -->
+
+# Bug Fix: [Component/Bug Name]
```
Comment (yodas7) [inline .github/PULL_REQUEST_TEMPLATE/bug_fixes.md:3]: I find it difficult to write this.

File: .github/PULL_REQUEST_TEMPLATE/bug_fixes.md (hunk 2 above)
Comment (hiro-daikin) [reply .github/PULL_REQUEST_TEMPLATE/bug_fixes.md:37]: @yodas7 @rinjInTokyo 
I think we can remove this part, and introduce a QA system where a team member can verify these changes

File: .github/PULL_REQUEST_TEMPLATE/bug_fixes.md (hunk 1 above)
Comment (hiro-daikin) [reply .github/PULL_REQUEST_TEMPLATE/bug_fixes.md:32]: @yodas7 I think you're right - I'll remove this section

**Review by yodas7 — APPROVED** [review]
Thank you!

**Review by rinjInTokyo — APPROVED** [review]
